ataos/Dockerfile

```diff
@@ -8,22 +8,22 @@ WORKDIR /home/saluser/repos
 RUN git clone https://github.com/lsst-ts/ts_ataos.git
 
 WORKDIR /home/saluser/repos/ts_ataos
-#RUN git fetch --all && git checkout "tags/v1.4.1" -b "v1.4.1"
-RUN git checkout tickets/DM-22700
-
-WORKDIR /home/saluser/repos/ts_xml
-#RUN git fetch --all && git checkout "tags/v1.4.1" -b "v1.4.1"
-RUN git fetch --all && git checkout tickets/DM-22700
-
-WORKDIR /home/saluser/repos/ts_config_attcs
-#RUN git fetch --all && git checkout "tags/v1.4.1" -b "v1.4.1"
-RUN git fetch --all && git checkout tickets/DM-22700
-
-WORKDIR /home/saluser
-RUN source /opt/lsst/software/stack/loadLSST.bash && setup lsst_distrib && \
-    source /home/saluser/repos/ts_sal/setup.env && \
-    setup ts_sal -t current && \
-    make_idl_files.py ATAOS ATMCS ATPneumatics ATHexapod ATCamera ATPtg
+RUN git fetch --all && git checkout "tags/v1.5.0" -b "v1.5.0"
+#RUN git checkout tickets/DM-22700
```

Comment:
Since it's commented out, can the following section just be removed?

Reply:
I like to keep this one here so I know where should I touch to flip from a tag to a branch. If you don't mind, I'd like to keep the commented line.

Reply:
That's fine for now. We need to revisit this in the not too distant future.